plugin/src/test/kotlin/org/jlleitschuh/gradle/ktlint/android/KtlintPluginAndroidTest.kt

```diff
@@ -88,11 +88,6 @@ class KtlintPluginAndroidTest : AbstractPluginTest() {
             minimumJava = 11,
             maximumJava = 17
         ),
-        MAX_GRADLE_MIN_AGP(
-            GradleVersion.version(TestVersions.maxSupportedGradleVersion),
-            TestVersions.minAgpVersion,
-            TestVersions.minSupportedKotlinPluginVersion
-        ),
```

Comment:
Why drop this?

Reply:
the old AGP doesnt work with Gradle 9, so this combination no longer works